packages/javascript/bh-shared-ui/src/components/ColumnHeaders/ColumnHeaders.tsx

```
@@ -81,7 +81,9 @@ export const SortableHeader: React.FC<SortableHeaderProps> = (props) => {
                     <TooltipProvider>
                         <TooltipRoot>
                             <TooltipTrigger>
-                                <FontAwesomeIcon className={cn('m-1')} size={'sm'} icon={faInfoCircle} />
+                                <div>
+                                    <FontAwesomeIcon className={cn('m-1')} size={'sm'} icon={faInfoCircle} />
+                                </div>
```

Do we need this? Should we make sure the mocked version of FontAwesomeIcon can accept a `ref` instead?

---

I was going to make a separate ticket for resolving this issue, since we already use this fix in a few other places and it may be worth cleaning up the tooltip component in doodle as part of the same effort. The intention here was just to suppress the error messages in the meantime